### PR TITLE
docker: update alpine and include notes.

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

**Comment by fguisso** [comment]
Tested with Docker for mac and every works fine.

if you can add a comment to remember if user don't want to preserve the `data` and `log` dirs, he need to use this basic config to put off from mounted volume:
```
rpcserver=
simnet=1
datadir=~/data/
logdir=~/logs/
```

File: Dockerfile.alpine [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
 RUN CGO_ENABLED=0 GOOS=linux GO111MODULE=on go install . ./cmd/...
 
 # Production image
-FROM alpine:3.6
+FROM alpine:3.10.1
 
 RUN apk add --no-cache ca-certificates
 COPY --from=0 /go/bin/* /bin/
 
 EXPOSE 9108
```
Comment (fguisso) [inline Dockerfile.alpine]: for a better understanding:
```
# Expose mainnet default ports [server rpc]
EXPOSE 9108 9109
```

File: Dockerfile.alpine [hunk 2]
```diff
@@ -1,17 +1,45 @@
...
+EXPOSE 9109
+EXPOSE 19108
```
Comment (fguisso) [inline Dockerfile.alpine]:
```
# Expose testnet default ports [server rpc]
EXPOSE 19108 19109
```

File: Dockerfile.alpine [hunk 3]
```diff
@@ -1,17 +1,45 @@
...
+EXPOSE 19109
+EXPOSE 18555
```
Comment (fguisso) [inline Dockerfile.alpine]:
```
# Expose simnet default ports [server rpc]
EXPOSE 18555 19556
```

**Comment by dajohi** [comment]
@fguisso addressed your review.

**Pull request merged**